bin/imsim.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python
```

Comment:
ok.  We can update the README with this info, though it would be good to get a proper fix for this.